badgecheck/util.py

```diff
@@ -6,6 +6,9 @@
 import requests_cache
 
 
+OPENBADGES_CONTEXT_URI_V2 = "https://w3id.org/openbadges/v2"
```

Comment:
Does it make sense to associate this with the product version and put in `version.py`? (If we did and just used `OPENBADGES_CONTEXT_URI` in the code, then it would neatly unbind it from a particular version of the context, but I'm not sure that's useful...